04_numeronym/lexual/numeronym.go

```diff
@@ -0,0 +1,29 @@
+package main
+
+import (
+	"fmt"
+	"io"
+	"os"
+	"strconv"
+)
+
+var out io.Writer = os.Stdout
+
+func numeronym(s string) string {
+	if len(s) <= 3 {
+		return s
+	}
+	return s[:1] + strconv.Itoa(len(s)-2) + s[len(s)-1:]
```

Comment:
You don't need to access an entire slice for a single element. `s[0] + ... + s[len(s)-1]` would do.
Since we are dealing with strings, the result of the `+` ends up the same anyway though, so all good.